README.md

```diff
@@ -6,7 +6,7 @@
 > Ethereum Client-VM Connector API
 
 The EVMC is the low-level ABI between Ethereum Virtual Machines (EVMs) and 
-Ethereum Clients. On the EVM-side it supports classic EVM1 and [eWASM].
+Ethereum Clients. On the EVM-side it supports classic EVM1 and [Ewasm].
```

Comment:
I think "EVM side" should be without dash

Reply:
I don't think there's an agreement on the name, can you just have the wasm stuff in a separate PR?